examples/acados_python/getting_started/test_ocp_setting.py

```diff
@@ -210,9 +238,11 @@
 ocp.solver_options.nlp_solver_tol_ineq = TEST_TOL
 ocp.solver_options.nlp_solver_tol_comp = TEST_TOL
 
-ocp.solver_options.qp_solver_cond_N = 10
-ocp.solver_options.nlp_solver_max_iter = 80
+ocp.solver_options.qp_solver_cond_N = int(N/2)
```

Comment:
maybe add a safeguards that this should be at least 1

Reply:
I mean, if N=1, `int(N/2)` is 0 or 1?

Reply:
this is just the test.
It is checked in the corresponding setter:
https://github.com/FreyJo/acados/blob/exact_hess_template/interfaces/acados_template/acados_template/acados_ocp.py#L1810